data/names/ru.json

```diff
@@ -3447,31 +3535,50 @@
       "Винсон",
       "Винтер",
       "Витале",
-      "Виттен",
       "Витт",
+      "Виттен",
       "Вишневски",
+      "Вишневский",
+      "Вмятина",
+      "Во",
       "Воган",
+      "Волк",
       "Вольф",
-      "Вонг",
+      "Воля",
+      "Вомак",
       "Вон",
+      "Вонг",
+      "Ворота",
       "Восс",
-      "Во",
+      "Восток",
```

Comment:
This seems to be a translation of the name [East](https://en.wikipedia.org/wiki/East_(surname\)) as a common noun, as in a cardinal direction. It would be transcribed Ист.